hbase-client/src/main/java/org/apache/hadoop/hbase/client/AsyncAdmin.java

```diff
@@ -1281,6 +1281,14 @@
    */
   CompletableFuture<Boolean> normalize();
```

Comment:
nit: could we add a default method here that calls `normalize(NormalizeTableFilterParams)` with a default one? seems like we could clean up some stuff in the various *Admin implementations and `RequestConverter`